server/graphql/v1/mutations/notifications.js

```diff
@@ -0,0 +1,94 @@
+import Promise from 'bluebird';
+import { values } from 'lodash';
+import models, { Op } from '../../../models';
+import activities from '../../../constants/activities';
+import { channels } from '../../../constants';
+import { Forbidden } from '../../errors';
+
+const NotificationPermissionError = new Forbidden({
+  message: "This notification does not exist or you don't have the permission to edit it.",
+});
+
+export async function editNotifications(args, remoteUser) {
+  if (!(remoteUser && remoteUser.isAdmin(args.id))) {
+    throw NotificationPermissionError;
+  }
+
+  if (!args.notifications) return Promise.resolve();
+  let newNotifications = [];
+
+  return models.Notification.findAll({ where: { CollectiveId: args.id, UserId: null } })
```

Comment:
As we're loading all notifications here, this is going to delete all existing notifications (for `email`, `twitter`, `gitter` and `slack`) that already exists for the collective.

Reply:
A simple way to fix this would be to rename this function `editWebhooks` and to make it deal only with webhooks.

Reply:
Addressed.